src/views/Home/components/EarnAssetCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Heading, Card, CardBody, ArrowForwardIcon } from '@pancakeswap-libs/uikit'
+import { NavLink } from 'react-router-dom'
+import pools from 'config/constants/pools'
+
+const StyledFarmStakingCard = styled(Card)`
+  background: linear-gradient(#53dee9, #7645d9);
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 344px;
+  width: 100%;
+  ${({ theme }) => theme.mediaQueries.lg} {
+    margin: 0;
+    max-width: none;
+  }
+`
+const Label = styled(Heading)`
+  color: ${({ theme }) => theme.colors.contrast};
+`
+const CardMidContent = styled(Heading).attrs({ size: 'xl' })`
+  color: ${({ theme }) => theme.colors.invertedContrast};
+  font-size: 35px;
```

Comment:
Same as above.

Plus, please use the font sizes from the theme. Figma has them at 44px and 24px respectively.

Reply:
font-size: 40px;
 line-height: 44px;